src/submcu.cpp

```diff
@@ -419,6 +419,31 @@ void SM_Opcode_LDX(uint8_t opcode) // a2, a6, ae, b6, be
     SM_Update_NZ(sm.x);
 }
 
+void SM_Opcode_LDY(uint8_t opcode) // a0, a4, ac, b4, bc
+{
+    uint8_t val = 0;
+    switch (opcode)
+    {
+        case 0xa0:
+            val = SM_ReadAdvance();
+            break;
+        case 0xa4:
+            val = SM_Read(SM_ReadAdvance());
+            break;
+        case 0xac:
+            val = SM_Read((SM_ReadAdvance() + sm.x) & 0xff);
+            break;
+        case 0xb4:
```

Comment:
0xb4 is zero page x addressing mode according to manual, so should be
`val = SM_Read((SM_ReadAdvance() + sm.x) & 0xff);`